modules/electrophysiology_browser/php/events.class.inc

```diff
@@ -89,53 +99,53 @@ class Annotations extends \NDB_Page
                 return (new \LORIS\Http\Response\JSON\BadRequest());
             }
 
-            (new ElectrophysioAnnotations(intval($parameters['physioFileID'])))
-                ->delete(intval($parameters['instance_id']));
+            // (new ElectrophysioEvents(intval($parameters['physioFileID'])))
+            //    ->deleteEvent(intval($parameters['instance_id']));
 
             return (new \LORIS\Http\Response\JSON\OK());
         case 'POST':
-            $parameters = json_decode((string) $request->getBody(), true);
-            if (!$user->hasPermission('electrophysiology_browser_edit_annotations')
+            // TODO: Better failure reporting
+            $parameters = json_decode((string)$request->getBody(), true);
+
+            if (!$user->hasPermission(
+                'electrophysiology_browser_edit_annotations'
```

Comment:
Rename permission?

Reply:
Will not be done at this time (see first mention)